RPG Wizardry/Assets/Scripts/ResearchData/DataStub/DataStubBin.cs

```diff
@@ -0,0 +1,114 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System.Collections.Generic;
+
+namespace nl.SWEG.RPGWizardry.ResearchData
+{
+    public class DataStubBin
+    {
+        public DataBin Bin = new DataBin(new List<Fragment>
+        {
+            new Fragment()
```

Comment:
Is there a nicer way to do this? (e.g. JSON)

Reply:
Added a Trello card for unloading/loading of JSON instead of using this datastub. When doing the JSON part we might as well try and save the solved Databin with.